### ISSUE TITLE
Simulating MonoCam with PhoSim

### ISSUE THREAD

**Issue by drphilmarshall** [DarkEnergyScienceCollaboration/Twinkles#256]
@DarkEnergyScienceCollaboration/twinkles 
Did you all see this email, to the SSim list, from @johnrpeterson ? Looks interesting, and we may be able to both contribute and learn something :-)

> On Thursday at 4 pm ET, we’ll be discussing simulating monocam with phosim.  Anyone is welcome.  Andrei will give an intro to monocam, and then we’ll informally think through some projects.

> Meeting info:
————————
PhoSim - MonoCam discussion meeting
Thursday June 9th at 4 pm ET

**Comment by drphilmarshall** [comment]
Thinking about this a little more: the big challenge with PhoSimulating MonoCam will be the telescope model, I guess. What we can maybe offer (already, and then again in the fall via Run 3), is MonoCam-like data, which may still be of interest to the SAWG group looking to compare real MonoCam images with simulated ones. Our DM Level 2 pipeline should be of interest to the MonoCam group too.

**Comment by drphilmarshall** [comment]
The MonoCam confluence pages are extensive, and give a good overview of what they are doing; I found [this page](https://confluence.slac.stanford.edu/display/LSSTDESC/MonoCam-I%3A+NOFS+May+2-11%2C+2016) to be most closely connected to Twinkles, because it describes the DM processing they have carried out. The MonoCam team take 300s exposures on a 1.5m telescope, so the atmosphere properties are different from LSST's, as well as the telescope properties. One of John's group is going to work on the PhoSim simulations. Our main contribution could be in helping further the DM pipeline; it's not clear that comparing MonoCam data with Twinkles data is interesting to the MonoCam team. Regarding the DM analysis, I left a comment for Simon and Merlin on the confluence page I just linked to, asking to see the code/recipe they have been following, and pointing them at our DM Level 2 recipe. Hopefully this will help connect us to the group!
